lib/elliptic/curve/base.js

```diff
@@ -243,6 +242,52 @@ BasePoint.prototype.validate = function validate() {
   return this.curve.validate(this);
 };
 
+BaseCurve.prototype.decodePoint = function decodePoint(bytes, enc) {
+  bytes = utils.toArray(bytes, enc);
+
+  if (this.type !== 'mont') {
```

Comment:
Yeah, I considered that. Didn't feel like it would buy much (at least yet)
and easier to review a simple move. You have any strong feelings either
way? I didn't actually try it on for size yet.